### ISSUE TITLE
High-N generation

### ISSUE THREAD

**Issue by vuhongai** [agemagician/Ankh#28]
Dear authors,
Thank you for open-sourcing your amazing PLM. 
Can you please provide an example of how to fine-tune model for sequence generation based on protein family?
Is that the same as fine-tuning causal language modeling with [transformers ](https://github.com/huggingface/transformers/tree/main/examples/pytorch/language-modeling) library? 

Thank you.
Ai

**Comment by hazemessamm** [comment]
Hello, please checkout this notebook for simple generation tutorial, https://github.com/agemagician/Ankh/blob/main/examples/simple_generation.ipynb

If you want to fine-tune you can train it using `transformers` library as a sequence to sequence task but the input sequence will just be masked using your masking function.

I will keep this issue open in case I did not answer your question and you need more elaboration.

**Comment by hazemessamm** [comment]
I will try to upload a simple notebook for fine-tuning soon too.

**Comment by vuhongai** [comment]
Thank you, it answers my question. I'll wait for your example.
